Issue #1323 (#1527)
MicrosoftDocs/OfficeDocs-SharePoint: OneDrive/sync-process.md

The previous OneDrive for Business sync app (Groove.exe) used a polling service to check for changes on a predetermined schedule. Polling can lead to system lag and slowness because it requires a lot of computing power. Using WNS is a significant enhancement.
  

## Auhentication protocols

> The authentications protocols depend Sharepoint version you are using

Reviewer: ```suggestion
The authentication protocols depend on which version of SharePoint you are using.
```

>  - For Sharepoint Server 2019, uses NTLM.

Reviewer: ```suggestion
 - Sharepoint Server 2019 uses NTLM.
```

>  - For Sharepoint Online, uses fed auth.

Reviewer: ```suggestion
 - SharePoint Online uses FedAuth.
```

## Security and encryption

File chunks are stored in multiple containers in Azure, each of which is given a unique key. Each key is required to reassemble the complete file. There’s also a separate master key encrypting each file chunk key, ensuring the data remain secure even when not moving.

> ## Related Seccion

Reviewer: ```suggestion
## Related topic
```

> - [Sharepoint Authentication](https://docs.microsoft.com/sharepoint/authentication)

Reviewer: ```suggestion
- [SharePoint Online Authentication](https://docs.microsoft.com/sharepoint/authentication)
```
